library/Hooks-windows.cpp

```diff
@@ -612,25 +613,22 @@ DFhackCExport int SDL_NumJoysticks(void)
 static void (*_SDL_GL_SwapBuffers)(void) = 0;
 DFhackCExport void SDL_GL_SwapBuffers(void)
 {
-    if(!inited)
-        FirstCall();
+    InitSDLPointers()
```

Comment:
semicolons seem to be missing.

Reply:
i was compiling on linux and did not notice that this file was 'windows' specific until i had already pushed. Sorry about that.